src/components/Insights/SentimentTreeview.js

```diff
@@ -212,7 +215,8 @@ export default class SentimentTreeview extends React.Component {
                 </Subheader>
                 <div style={styles.searchBox}>
                    { <TypeaheadSearch 
-                        dashboardRefreshFunc={(maintopic, conjunctivetopics)=>this.handleDataFetch(maintopic, conjunctivetopics)}
+                        dashboardRefreshFunc={this.handleDataFetch}
```

Comment:
bind `handleDataFetch`

Reply:
This is not necessary given that handleDataFetch now is an arrow function.